### ISSUE TITLE
Error while importing model

### ISSUE THREAD

**Issue by MARK2580** [LendoK/Blender_GTA_V_model_importer#11]
![Nh45XHowDI](https://user-images.githubusercontent.com/11617079/92330238-0346cc00-f076-11ea-8457-64716e0af1f7.jpg)

Model name ch_chint02_beam_ceiling_shell from casino heist update. I think you have gta v / OpenIV to extract this model and check...

**Comment by ZackFair00210** [comment]
Yes!! exactly! I`ve been wrecking my head for this!! please tell me how do I get rid of this!!?

**Comment by LendoK** [comment]
Can you please tell me the exact path where to find the model "ch_chint02_beam_ceiling_shell", or attache the file here?

**Comment by ZackFair00210** [comment]
uh??

**Comment by ZackFair00210** [comment]
What do you mean? its your addon tool right, and which model?

**Comment by LendoK** [comment]
Yes, I'm trying to reproduce this bug. I need a model causing this error, but I con not find "ch_chint02_beam_ceiling_shell". The OpenV search is not helpful

**Comment by ZackFair00210** [comment]
Okay I`m providing you my own model, and let me know if you find the error
>
https://www.deviantart.com/users/outgoing?https://mega.nz/file/Ya5VTQ6J#Q47zAMxP_wlrghD8s9HYJklOEjD3kW56xkXXSuaHJ_0
I dunno what happens to the addon, when I try to import the odr or the odd
files.

**Comment by LendoK** [comment]
@ZackFair00210 I was able to fix some problems with the skeleton import. Hope it helps. But none of the errors are related to the vertex attribute error @MARK2580  reported

**Comment by ZackFair00210** [comment]
ah..I see soo...now what?
